packages/client/src/components/Modal/Modal.tsx

```diff
@@ -6,45 +6,44 @@ import {
   StModalCloseIcon,
 } from './style';
 import { StButtonCloseModal } from 'components/Button/style';
+import { CSSProp } from 'styled-components';
 
 interface ModalType {
   children?: ReactNode;
   title?: string;
   handleCloseModal?: () => void;
-  width?: number;
-  verticalPaddings?: number;
-  horizontalPaddings?: number;
+  styles?: CSSProp;
+  isCloseOutside?: boolean;
+  isCrossButton?: boolean;
 }
 
 const Modal = (props: ModalType) => {
   const {
     title,
     handleCloseModal,
-    width,
-    verticalPaddings,
-    horizontalPaddings,
+    isCloseOutside,
```

Comment:
Что означает `isCloseOutside`?

Reply:
можно ли закрыть модалку по клику вне или нет

Reply:
"можно ли", звучит как **can**, например:
```suggestion
    canBeClosedOutside,
```